### PR TITLE
Proto for dynamic currencies

### PR DESCRIPTION
**Pull request by coddmeistr**
Linked PR: https://github.com/slntopp/nocloud/pull/1653

### CODE REVIEW

File: billing/billing.proto [hunk 1]
```diff
@@ -548,7 +558,7 @@ service CurrencyService {
   rpc GetExchangeRate (GetExchangeRateRequest)
       returns (GetExchangeRateResponse) {
     option (google.api.http) = {
-      get: "/billing/currencies/rates/{from}/{to}"
+      get: "/billing/currencies/rates/{from.name}/{to.name}"
```
Comment (gorobot-nz) [inline billing/billing.proto:561]: Why did you rename the variable?

Comment (coddmeistr) [reply billing/billing.proto:561]: Since I changed "Currency" type from primitive to Message I had to change URL mappings. But yeah, my example will not work, so I either move those mappings from URL Path to URL queries or to Body, which one should I choose?
